site/layouts/_default/single.html

```diff
@@ -7,16 +7,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="Description">
     <link rel="stylesheet" href="site.css">
-    <link href="https://code.cdn.mozilla.net/fonts/fira.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500&display=swap" rel="stylesheet">
```

Comment:
This and the linked fonts should all be checked into the repo so this works entirely over IPFS without external resources.